nobl9/resource_alertmethod_test.go

```diff
@@ -28,7 +28,7 @@ func TestAcc_Nobl9AlertMethod(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			resource.Test(t, resource.TestCase{
+			resource.ParallelTest(t, resource.TestCase{
```

Comment:
this might be a problem because in tests we were veeeery lazy and reused for example `testAlertPolicyWithoutIntegration` from `resource_alert_policy_test.go` in `resource_slo_test.go`

Reply:
yah, i already ran into issues, and reverted in the other branch with the tests.

Reply:
I can revert here too if you want